packages/ipa-certified-attributes-importer/src/index.ts

```diff
@@ -44,8 +44,18 @@ const loggerInstance = logger({
   correlationId,
 });
 
-function toKey<T>(a: T): string {
-  return JSON.stringify(a);
+function toTenantKey(key: {
+  origin: string | undefined;
+  value: string | undefined;
+}): string {
+  return JSON.stringify(key);
+}
+
+function toAttributeKey(key: {
+  origin: string | undefined;
+  code: string | undefined;
+}): string {
+  return JSON.stringify(key);
```

Comment:
What about something like this 👇 
So that the functions prevent passing wrong arguments e.g., exchanging origin and code by design.

```ts
function toTenantKey(tenant: Tenant): string {
  return JSON.stringify(t.externalId);
}

function toAttributeKey(attribute: Attribute): string {
  return JSON.stringify({ origin: a.origin, code: a.code });
}
```

Reply:
the problem is that I don't always have a Tenant or an Attribute. For the AttributeKey could be something like `Pick<Attribute, "origin" | "code">` for the TenantKey is more difficult. Sometimes I have the externalId, and sometimes the origin and the value directly.

Reply:
Oh ok, it's not so straightforward - feel free to leave it as it is 👍